### DISCUSSION TITLE
SSL Certificate - help with renewal?

### DISCUSSION THREAD

**Discussion by ianatinsight** [amidaware/tacticalrmm#1448] [Q&A]
Hey there!

I apologize if this isn't the correct place to submit this question. I am not very familiar with SSL Certificates and Let's Encrypt. I have recieved the warning that the certificate is going to expire in a few days.

I played around with it a little- but wasn't able to make any progress. I think that the guides I found online may not apply to this situation.

So, I'm hoping that I may be able to get some assistance or possibly be pointed in the right direction to renew the cert?

Thank you very much for reading. :-)

**Comment by dinger1986** [comment]
You should just need to run through https://docs.tacticalrmm.com/update_server/#keeping-your-lets-encrypt-ssl-certificate-up-to-date

**Reply by ianatinsight** [reply]
Ah! Thank you. I'm so grateful for your helpful and quick response!

**Comment by silversword411** [comment]
If you're still having problems and just want us to do it for you just open a ticket and we'll be happy to assist.

https://support.amidaware.com/

**Reply by ianatinsight** [reply]
Thank you so much! I really appreciate your assistance.
